ui/js/controllers/perf/graphs.js

```diff
@@ -653,6 +665,17 @@ perf.controller('GraphsCtrl', [
                     $scope.seriesList = [];
                     addSeriesList([]);
                 }
+                if ($stateParams.tooltip) {
+                    var tooltipString = decodeURIComponent($stateParams.tooltip).replace(/[\[\]"]/g, '');
+                    var tooltipArray = tooltipString.split(",");
+                    var tooltip = {
+                            projectName: tooltipArray[0],
+                            signature: tooltipArray[1],
+                            resultSetId: parseInt(tooltipArray[2]),
+                            jobId: parseInt(tooltipArray[3])
```

Comment:
I can think of cases where we might not care about the specified job id, so we might want to support the cases where the user only provides the first three variables. For this, we can do something like:

    jobId: (tooltipArray[3] !== undefined) ? parseInt(tooltipArray[3]) : null;

Of course you will need to handle null values for jobId elsewhere, but that shouldn't be too hard.